Example/Source/Examples/Images/ImagesInCollectionBrickHorizontalViewController.swift

```diff
@@ -33,7 +33,7 @@ class ImagesInCollectionBrickHorizontalViewController: BrickViewController {
         collectionSection.repeatCountDataSource = self
 
         let section = BrickSection(bricks: [
-            CollectionBrick(scrollDirection: .Horizontal, dataSource: CollectionBrickCellModel(section: collectionSection, configureHandler: { (cell) in
+            CollectionBrick(scrollDirection: .Horizontal, dataSource: CollectionBrickCellModel(section: collectionSection, registerBricksHandler: { (cell) in
                 cell.brickCollectionView.registerBrickClass(ImageBrick.self)
```

Comment:
As you are registering just the class, shouldn't you add these brickTypes in the initializer of the CollectionBrick?